From patches Mon Sep 17 00:00:00 2001
From: Igor Mammedov <imammedo@redhat.com>
To: Igor Mammedov <imammedo@redhat.com>, qemu-devel@nongnu.org
Cc: Anthony Liguori <aliguori@us.ibm.com>, ehabkost@redhat.com,
 qemu-devel@nongnu.org, Peter Maydell <peter.maydell@linaro.org>,
 anthony.perard@citrix.com, pbonzini@redhat.com, afaerber@suse.de
Date: Tue, 30 Apr 2013 15:41:24 +0200
Message-id: 1367329288-27178-2-git-send-email-imammedo@redhat.com
Subject: [PATCH 1/5] add hot_add_cpu hook to QEMUMachine

Hook should be set by target that implements
CPU hot-add via cpu-add QMP command.

Signed-off-by: Igor Mammedov <imammedo@redhat.com>
Reviewed-by: Eduardo Habkost <ehabkost@redhat.com>
---
 include/hw/boards.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/hw/boards.h b/include/hw/boards.h
index 425bdc7..75cd127 100644
--- a/include/hw/boards.h
+++ b/include/hw/boards.h
@@ -43,6 +43,7 @@ typedef struct QEMUMachine {
     GlobalProperty *compat_props;
     struct QEMUMachine *next;
     const char *hw_version;
+    void (*hot_add_cpu)(const int64_t id, Error **errp);
 } QEMUMachine;
 
 int qemu_register_machine(QEMUMachine *m);
-- 
1.7.1

